git-remote-gosh/Cargo.toml

```diff
@@ -47,7 +47,7 @@ indexmap = '^1'
 lazy_static = '^1'
 log = '^0'
 log4rs = '^1'
-lru = '^0'
+lru = '^0.8.0'
```

Comment:
It is required. Otherwise it takes different versions on different os'es. (pre 0.8 for macos and 0.8 on windows)